### PR TITLE
App configuration Docs Update

### PR DESCRIPTION
**Pull request by mrm9084**

### CODE REVIEW

**Review by saragluna — CHANGES_REQUESTED** [review]

File: sdk/appconfiguration/azure-spring-cloud-starter-appconfiguration-config/README.md [hunk 1]
```diff
@@ -111,37 +137,36 @@ INFO 17496 --- [TaskScheduler-1] o.s.c.e.event.RefreshEventListener       : Refr
...
 
-#### Push Based Refresh
+##### Push Based Refresh
 
-The Web Provider can be connect to your Azure App Configuration store via an Azure Event Grid Web Hook to trigger a refresh event. By adding the Spring Actuator as a dependency you can add App Configuration Refresh as an exposed endpoint. There are two options appconfiguration-refresh and appconfiguration-refresh-bus. These endpoints work just like there counterparts refresh and refresh-bus, but have the required web hook authorization to work with Azure Event Grid.
+The Web Provider can be connect to your Azure App Configuration store via an Azure Event Grid Web Hook to trigger a refresh event. By adding the Spring Actuator as a dependency you can add App Configuration Refresh as an exposed endpoint. There are two options appconfiguration-refresh and appconfiguration-refresh-bus. These endpoints work just like there counterparts refresh and refresh-bus, but have the required web hook authorization to work with Azure Event Grid. When refreshing multiple instances `azure-servicebus-jms-spring-boot-starter` needs to be setup.
```
[the remark this answers is not in the document]
Comment (mrm9084) [reply sdk/appconfiguration/azure-spring-cloud-starter-appconfiguration-config/README.md:154]: I mean the configurations need to be set for servicebus. We will have an ms docs doc on how to do it.

[the remark this answers is not in the document]
Comment (saragluna) [reply sdk/appconfiguration/azure-spring-cloud-starter-appconfiguration-config/README.md:154]: Do we need to change the description here now? Setting up multiple instances of jms starter doesn't make much sense to me.

[the remark this answers is not in the document]
Comment (mrm9084) [reply sdk/appconfiguration/azure-spring-cloud-starter-appconfiguration-config/README.md]: @saragluna Tried to fix the wording. This works the same as the Spring Bus Refresh.

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**